libretroshare/src/gossipdiscovery/gossipdiscoveryitems.h

```diff
@@ -31,79 +31,77 @@
 
 #include "serialiser/rsserializer.h"
 
-const uint8_t RS_PKT_SUBTYPE_DISC_PGP_LIST           = 0x01;
-const uint8_t RS_PKT_SUBTYPE_DISC_PGP_CERT           = 0x02;
-const uint8_t RS_PKT_SUBTYPE_DISC_CONTACT_deprecated = 0x03;
-const uint8_t RS_PKT_SUBTYPE_DISC_SERVICES           = 0x04;
-const uint8_t RS_PKT_SUBTYPE_DISC_CONTACT            = 0x05;
-const uint8_t RS_PKT_SUBTYPE_DISC_IDENTITY_LIST      = 0x06;
+enum class RsGossipDiscoveryItemType : uint8_t
+{
+	PGP_LIST           = 1,
```

Comment:
Ok I can change that to my eyes it is equivalent ;)
I used decimal notation because soon we will have tons of JavaScript kiddies reading our code as they fiddle with JSON API and lot of them doesn'T even know that when a number start with 0x it is to be read as hexadecimal, sad to say but it is true :(

Reply:
Very sad :D